packages/near-fast-auth-signer-e2e-tests/test-app/components/SignMultiChain.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { FormEvent } from 'react';
+
+type SignMultiChainProps = {
+  // eslint-disable-next-line no-unused-vars
+  onSubmitForm: (values: {
+    keyType: string,
+    chainValue: number,
+    amount: number,
+    chainId: string,
+    address: string}) => void
+}
+export default function SignMultiChain(props: SignMultiChainProps) {
+  const { onSubmitForm } = props;
+  const onSubmit = (event: FormEvent<HTMLFormElement>) => {
```

Comment:
I think we are using `react-hook-form` and `yup` for form management on `fast-auth-signer`, no? 

If so, can you please re-use it here to make it consistent with the codebase?

Reply:
Done